docs/topics/api/addons.rst

```diff
@@ -48,6 +48,7 @@ This endpoint allows you to search through public add-ons.
     :query boolean featured: Filter to only featured add-ons.  Only ``featured=true`` is supported.
         If ``app`` is provided as a parameter then only featured collections targeted to that application are taken into account.
         If ``lang`` is provided then only featured collections targeted to that language, (and collections for all languages), are taken into account. Both ``app`` and ``lang`` can be provided to filter to addons that are featured in collections that application and for that language, (and for all languages).
+    :query string guid: Filter by exact add-on guid. Multiple guids can be specified, separated by comma(s), in which case any add-ons matching any of the guids will be returned.  As guids are unique there should be at most one add-on result per guid specified..
```

Comment:
Double dot at the end and a double-space before the "As guids"

Reply:
double spaces are all over the docs and code comments.  (😛 #DoubleSpace4Life)